docs/bigtable-data-api.rst

```diff
@@ -0,0 +1,328 @@
+Data API
+========
+
+After creating a :class:`Table <gcloud.bigtable.table.Table>` and some
+column families, you are ready to store and retrieve data.
+
+Cells vs. Columns vs. Column Families
++++++++++++++++++++++++++++++++++++++
+
+* As we saw before, a table can have many column families.
+* As we'll see below, a table also has many rows (specified by row keys).
+* Within a row, data is stored in a cell. A cell simply has a value (as
+  bytes) and a timestamp. The number of cells in each row can be
+  different, depending on what was stored in each row.
+* Each cell lies in a column (**not** a column family). A column is really
+  just a more **specific** modifier within a column family. A column
+  can be present in every column family, in only one or anywhere in between.
+* Within a column family there can be many columns. For example within
```

Comment:
I link to it in `bigtable-table-api.rst`, do you think I just add a parenthetical at the end of this bullet?

```
...
  These would typically be represented as ``foo:bar`` and ``foo:baz``. (For
  more on column/column family semantics in Cloud Bigtable check out the
  `column families`_ doc.)

.. _column families: https://cloud.google.com/bigtable/docs/schema-design#column_families_and_column_qualifiers
```